### ISSUE TITLE
Contract dependency not found (on windows)

### ISSUE THREAD

**Issue by mriccobene** [nomiclabs/truffle-flattener#21]
We have this directory organization:

```
root
 |- contracts
      |- Test.sol
 |- node_modules
      |- zeppelin-solidity
```
      
Our contract Test.sol has 

`
import "zeppelin-solidity/contracts/ownership/Ownable.sol";
`

Starting 

`
truffle-flattener contracts/Test.sol
`

we got:

`
Could not find ..\ownership\Ownable.sol from any sources
`

**Comment by mriccobene** [comment]
Yes!

**Comment by mriccobene** [comment]
```
truffle.cmd compile contracts/Test.sol
Compiling .\contracts\Migrations.sol...
...
Compiling zeppelin-solidity/contracts/ownership/Ownable.sol...
...
```

**Comment by alcuadrado** [comment]
Can you paste the contents of `contracts/Test.sol` here please?

**Comment by mriccobene** [comment]
No. But I prepared a fake Test.sol. 

This file compiles fine:

```
>truffle.cmd compile contracts/Test.sol
Compiling .\contracts\Test.sol...
Compiling zeppelin-solidity/contracts/math/SafeMath.sol...
Compiling zeppelin-solidity/contracts/ownership/Ownable.sol...
Compiling zeppelin-solidity/contracts/token/ERC20/BurnableToken.sol...
Compiling zeppelin-solidity/contracts/token/ERC20/DetailedERC20.sol...
Compiling zeppelin-solidity/contracts/token/ERC20/MintableToken.sol...
Compiling zeppelin-solidity\contracts\math\SafeMath.sol...
Compiling zeppelin-solidity\contracts\ownership\Ownable.sol...
Compiling zeppelin-solidity\contracts\token\ERC20\BasicToken.sol...
Compiling zeppelin-solidity\contracts\token\ERC20\ERC20.sol...
Compiling zeppelin-solidity\contracts\token\ERC20\ERC20Basic.sol...
Compiling zeppelin-solidity\contracts\token\ERC20\StandardToken.sol...
Writing artifacts to .\build\contracts
```

But truffle-flattener reports this error:

```
>truffle-flattener contracts/Test.sol
Error: Could not find BasicToken.sol from any sources
    at C:\Users\miche\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\truffle-resolver\index.js:76:23
    at C:\Users\miche\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\async\internal\onlyOnce.js:12:16
    at next (C:\Users\miche\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\async\whilst.js:68:18)
    at C:\Users\miche\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\truffle-resolver\index.js:64:7
    at C:\Users\miche\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\truffle-resolver\fs.js:85:5
    at C:\Users\miche\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\async\internal\once.js:12:16
    at replenish (C:\Users\miche\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\async\internal\eachOfLimit.js:59:25)
...
```

Here is the content:
```
pragma solidity 0.4.19;

import "zeppelin-solidity/contracts/math/SafeMath.sol";
import "zeppelin-solidity/contracts/ownership/Ownable.sol";
import "zeppelin-solidity/contracts/token/ERC20/BurnableToken.sol";
import "zeppelin-solidity/contracts/token/ERC20/DetailedERC20.sol";
import "zeppelin-solidity/contracts/token/ERC20/MintableToken.sol";

contract Test is Ownable, DetailedERC20, MintableToken, BurnableToken {
    using SafeMath for uint256;

    function Test(
        string _name,
        string _symbol,
        uint8 _decimals
    )
    DetailedERC20 (_name, _symbol, _decimals) public
    {

    }
}
```

**Comment by liuhuanhui** [comment]
Do I have to solve this problem, too?

**Comment by alcuadrado** [comment]
Hi @liuhuanhui! I'm sorry, but i think I don't understand your message. Are you willing to fix this? It'd be awesome :)

**Comment by liuhuanhui** [comment]
hi @alcuadrado, Sorry, I just encountered this problem. I don't know how to solve it now.

**Comment by jakkyo** [comment]
Probably issue caused by dependency truffle-resolve: https://github.com/trufflesuite/truffle-resolver/blob/e113b9c76de588f8bf14a4355722965720b0ae31/index.js#L70

I found there its error message generating code: `var message = "Could not find " + import_path + " from any sources";`

**Comment by nicholaspai** [comment]
hi was this ever resolved? in my project i import open zeppelin's proxy contracts via

`import "zos-lib/contracts/upgradeability/UpgradeabilityProxy.sol";`

which imports itself via relative path:

`import './Proxy.sol';`
(absolute path: `zos-lib/contracts/upgradeability/Proxy.sol`)

and running into the same errors as @mriccobene : `Error: Could not find Proxy.sol from any sources`

**Comment by nandod** [comment]
Same problem here with 1.2.9 on Windows.

**Comment by alexanderm39** [comment]
Hi
I've created a pull request that fixes the issue:
[https://github.com/nomiclabs/truffle-flattener/pull/33](url)

**Comment by nihalmpatel** [comment]
Having the same issue while flattening a file!

Logs:
```
Error: Could not find IERC20.sol from any sources
    at C:\Users\Nihal Patel\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\truffle-resolver\index.js:76:23
    at C:\Users\Nihal Patel\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\async\internal\onlyOnce.js:12:16
    at next (C:\Users\Nihal Patel\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\async\whilst.js:68:18)
    at C:\Users\Nihal Patel\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\truffle-resolver\index.js:64:7
    at C:\Users\Nihal Patel\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\truffle-resolver\fs.js:85:5
    at C:\Users\Nihal Patel\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\async\internal\once.js:12:16
    at replenish (C:\Users\Nihal Patel\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\async\internal\eachOfLimit.js:61:25)
    at iterateeCallback (C:\Users\Nihal Patel\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\async\internal\eachOfLimit.js:50:17)
    at C:\Users\Nihal Patel\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\async\internal\onlyOnce.js:12:16
    at ReadFileContext.callback (C:\Users\Nihal Patel\AppData\Roaming\npm\node_modules\truffle-flattener\node_modules\truffle-resolver\fs.js:81:14) 'Error: Could not find IERC20.sol from any sources\n    at C:\\Users\\Nihal Patel\\AppData\\Roaming\\npm\\node_modules\\truffle-flattener\\node_modules\\truffle-resolver\\index.js:76:23\n    at C:\\Users\\Nihal Patel\\AppData\\Roaming\\npm\\node_modules\\truffle-flattener\\node_modules\\async\\internal\\onlyOnce.js:12:16\n
  at next (C:\\Users\\Nihal Patel\\AppData\\Roaming\\npm\\node_modules\\truffle-flattener\\node_modules\\async\\whilst.js:68:18)\n    at C:\\Users\\Nihal Patel\\AppData\\Roaming\\npm\\node_modules\\truffle-flattener\\node_modules\\truffle-resolver\\index.js:64:7\n    at C:\\Users\\Nihal Patel\\AppData\\Roaming\\npm\\node_modules\\truffle-flattener\\node_modules\\truffle-resolver\\fs.js:85:5\n    at C:\\Users\\Nihal Patel\\AppData\\Roaming\\npm\\node_modules\\truffle-flattener\\node_modules\\async\\internal\\once.js:12:16\n    at replenish (C:\\Users\\Nihal Patel\\AppData\\Roaming\\npm\\node_modules\\truffle-flattener\\node_modules\\async\\internal\\eachOfLimit.js:61:25)\n    at iterateeCallback (C:\\Users\\Nihal Patel\\AppData\\Roaming\\npm\\node_modules\\truffle-flattener\\node_modules\\async\\internal\\eachOfLimit.js:50:17)\n    at C:\\Users\\Nihal Patel\\AppData\\Roaming\\npm\\node_modules\\truffle-flattener\\node_modules\\async\\internal\\onlyOnce.js:12:16\n    at ReadFileContext.callback (C:\\Users\\Nihal Patel\\AppData\\Roaming\\npm\\node_modules\\truffle-flattener\\node_modules\\truffle-resolver\\fs.js:81:14)'
```

**Comment by mriccobene** [comment]
PR #33 solves the issue for me.

**Comment by alcuadrado** [comment]
Thanks @mriccobene, I'm merging it :)

**Comment by alcuadrado** [comment]
I just released version 1.2.10 which should work on windows.
